### ISSUE TITLE
Fast Forward Select+R also raises volume 

### ISSUE THREAD

**Issue by GetTuh** [cizia64/CrossMix-OS#87]
Every time i toggle FF with Select+R the volume goes up one notch. Present at least in SwanStation, NeoGeo, fake-08. Even in wrappers like pico8 where FF isn't supported select+R raises the volume.

**Comment by GetTuh** [comment]
Also Select+L lowers the volume.

**Comment by cizia64** [comment]
Yes I know but it was the best solution I had ;) 
Select + vol+ or vol- are shortcut managed by the keymon binary. Lile Start + vol+/vol- for brightness.
I have asked to TrimUI to change the behavior of the menu key because ideally I would like to use it as hotkey so...let's hope to have a better choice in the future.

**Comment by GetTuh** [comment]
Maybe then let's lower the volume after the hotkey is pressed? Not a perfect solution but I don't see a better one

**Comment by cizia64** [comment]
Yes it could be a workaround... However we should have good news from TrimUI about this problem in the next firmware, patience (I have no release date) 😉
